Apps/phslack/slack_consts.py

```diff
@@ -115,6 +115,7 @@
 SLACK_ERR_QUESTION_RESPONSE_NOT_AVAILABLE = "Response to question not available"
 SLACK_ERR_NO_RESPONSE_FROM_SERVER = "Got no response from the Slack server"
 SLACK_ERR_INVALID_CHANNEL_TYPE = "Please provide a valid value in the 'channel_type' action parameter"
+SLACK_ERR_LENGTH_LIMIT_EXCEEDED = "Based on your asset_id length ({asset_id}), valid length for the 'confirmation' parameter is {valid_length}"
```

Comment:
can we rename `asset_id` to `asset_length`? the former name is misleading since it's not actually the asset ID

Reply:
Done